torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -187,53 +187,41 @@ def _get_existing_inline_assertions(
                 continue
 
             compare_op = compare_arg.target
-            maybe_symint_arg, compare_int = compare_arg.args
-
-            # x >= 0 will sometimes be canonicalized to -x <= 0, so in some
-            # cases the operation before the comparison is to multiply by -1. We
-            # can undo the canonicalization here
-            if (
-                maybe_symint_arg.op == "call_function" and
-                maybe_symint_arg.target == operator.mul and
-                maybe_symint_arg.args[0] == -1
-            ):
-                maybe_symint_arg = maybe_symint_arg.args[1]
-                compare_op = operator.ge
-                compare_int = -1 * compare_int
+            lhs, rhs = compare_arg.args
```

Comment:
@avikchaudhuri this needed quite a rewrite, but I think that now the code is much cleaner. PTAL.

Reply:
We are actually deprecating this pass very soon in favor of deferred runtime assertion pass. So as long as you make the test pass, i think it is ok to do whatever here.